bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -5182,6 +5182,22 @@ void ReplicaImp::onExecutionFinish() {
     // We are about to stop execution. To avoid VC we now clear all pending requests
     clientsManager->clearAllPendingRequests();
   }
+
+  // Sending noop commands to get the system into a stable checkpoint,
+  // allowing the create dbCheckpoint operator command to create a dbCheckpoint/snapshot.
+  if (getReplicaConfig().dbCheckpointFeatureEnabled && isCurrentPrimary() &&
+      (DbCheckpointManager::instance().getNextStableSeqNumToCreateSnapshot().has_value() &&
+       DbCheckpointManager::instance().getNextStableSeqNumToCreateSnapshot().value() >= lastExecutedSeqNum)) {
+    bool noop =
```

Comment:
bool noop = DbCheckpointManager::instance().getNextStableSeqNumToCreateSnapshot().value() == (lastExecutedSeqNum + 1);

This is better, we donot need :? operator here.

Reply:
Fixed